importer/tests/test_views.py

```diff
@@ -66,5 +66,4 @@ def test_invalid_form(self):
         )
         self.assertTrue(resp.status_code == HTTP_OK)
         body = resp.content.decode("utf-8")
-        log.debug(body)
```

Comment:
nothing wrong with leaving this in.  They only show when that test fails and they are handy to have